File: Python/Program/Clothing.py
from Produk import Produk

class Clothing(Produk):
    __size = ""
    __material = ""
    __gender = ""
    
    def __init__ (self, size = "", material = "", gender = ""):
        self.__size = size
        self.__material = material
        self.__gender = gender
        
    # getter #
    def getSize(self):
        return self.__size

    def getMaterial(self):
        return self.__material

    def getGender(self):
        return self.__gender
        
    # setter #
    
    def setSize(self, size):
        self.__size = size

    def setMaterial(self, material):
        self.__material= material

    def setGender(self, gender):
        self.__gender = gender

File: PHP/Program/Clothing.php
<?php

require('Produk.php');

class Clothing extends Produk
{
	private $size, $material, $gender;
	
	public function __construct($size = '', $material ='', $gender = '')
	{
		$this->size = $size;
		$this->material= $material;
		$this->gender = $gender;
	}
	
	public function setSize($size)
	{
		$this->size = $size;
	}
	
	public function setMaterial($material)
	{
		$this->material = $material;
	}
	
	public function setGender ($gender)
	{
		$this->gender= $gender;
	}
	
	public function getSize()
	{
		return $this->size;
	}
	
	public function getMaterial()
	{
		return $this->material;
	}
	
	public function getGender()
	{
		return $this->gender;
	}
	
}
